sphinx/source/screens.rst

```diff
@@ -40,11 +40,44 @@ in Python.
 
 Screens are updated at the start of each interaction, and each time an
 interaction is restarted. Note that a ``with None`` statement does not
-cause an interaction to happen, and hence won't update a screen.
+cause an interaction to happen, and hence won't update a screen. A
+``with Pause(0)`` will be enough to trigger one, if necessary.
+
+A screen has a scope associated with it, giving values to some variables. There are different kinds
+of variables, in screens, which are resolved as shown in the following list:
+
+- First are local variables. These only exist in a screen that is being included in another with
+  the :ref:`use <sl-use>` statement instead of being shown (or called) on its own. They are very
+  similar with screen variables (see below), and created the same way, except that local variables
+  can only be accessed by the used screen. Those are set using :class:`SetLocalVariable`, among
+  other actions, or by a python block or line in the used screen. Actions such as
+  :class:`SetScreenVariable` will *not work* on local variables.
+    - Local parameters are the parameters taken by the used screen. They live in the same scope as
```

Comment:
Is there some reason this list isn't lined up?

Reply:
Yes, screen parameters are a subcategory of screen variables with particular behaviors